CliFx/Domain/CommandArgumentSchema.cs

```diff
@@ -176,4 +159,71 @@ internal partial class CommandArgumentSchema
                 BindingFlags.Public | BindingFlags.Static,
                 null, new[] {typeof(string), typeof(IFormatProvider)}, null);
     }
+
+    // Default and valid value handling.
+    internal partial class CommandArgumentSchema
+    {
+        public IReadOnlyList<string> GetValidValues()
+        {
+            var result = new List<string>();
+
+            // Some arguments may have this as null due to a hack that enables built-in options
+            if (Property == null)
+                return result;
+
+            var underlyingPropertyType =
+                Property.PropertyType.GetNullableUnderlyingType() ?? Property.PropertyType;
+
+            // Enum
+            if (underlyingPropertyType.IsEnum)
+                result.AddRange(Enum.GetNames(underlyingPropertyType));
+
+            return result;
+        }
+
+        public string? GetDefaultValue(object? instance)
```

Comment:
Let's narrow down the type of instance to `ICommand` so it's more clear.

Reply:
Fixed.